config/locales/en.yml

```diff
@@ -590,7 +590,6 @@ en:
         title: "Invoice Settings"
         enable_invoices?: "Enable Invoices?"
         invoice_style2?: "Use the alternative invoice model that includes total tax breakdown per rate and tax rate info per item (not yet suitable for countries displaying prices excluding tax)"
-        enable_receipt_printing?: "Show options for printing receipts using thermal printers in order dropdown?"
```

Comment:
Is it enough to remove the translations from the `en` file? Will the current worklow with transifex remove these from other locales? Thanks!